app/components/frame-viewer.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import assert from 'assert';
+import { mount } from 'enzyme';
+import FrameViewer from './frame-viewer';
+import FrameAnnotator from '../classifier/frame-annotator';
+
+const subject = {
+  locations: [{ image: 'www.testimage.com' }]
+};
+
+const videoSubject = {
+  locations: [{ video: 'www.testvideo.com' }]
```

Comment:
Similarly, we probably should have a mock data video and text subject for use with the dev classifier.